scripts/lint-packages.sh

```diff
@@ -477,6 +477,17 @@ lint_package() {
 			fi
 			unset urls_ok
 
+			# Check for GitHub archive URLs that should use /archive/refs/tags/
+			for url in "${TERMUX_PKG_SRCURL[@]}"; do
+				if [[ "$url" =~ github\.com/.*/archive/[^/]+\.(tar\.gz|zip)$ ]] && \
+				   [[ ! "$url" =~ /archive/refs/ ]]; then
```

Comment:
Mmmh didn't catch the mixed indentation here.
You can either leave this split across two lines with something like this.

```suggestion
				if [[ "$url" =~ github\.com/.*/archive/[^/]+\.(tar\.gz|zip)$ ]] &&
					! "$url" =~ /archive/refs/ ]]; then
```
Or just put it on a single line.
Either is fine, but we try to avoid mixed indentation unless required